internal/pkg/deploy/cloudformation/stack/backend_svc.go

```diff
@@ -152,6 +152,7 @@ func (s *BackendService) Template() (string, error) {
 		Secrets:                  convertSecrets(s.manifest.BackendServiceConfig.Secrets),
 		Aliases:                  aliases,
 		HTTPSListener:            s.httpsEnabled,
+		HTTPRedirect:             s.httpsEnabled,
```

Comment:
gotcha, so we're only supporting this field for now for LBWS as there is no need of a cdn for internal ALBs 👍